server/command_test.go

```diff
@@ -1,1088 +1,951 @@
 package main
 
 import (
+	"fmt"
 	"testing"
 	"time"
 
-	"github.com/mattermost/mattermost-plugin-msteams/server/metrics"
-	mockMetrics "github.com/mattermost/mattermost-plugin-msteams/server/metrics/mocks"
 	"github.com/mattermost/mattermost-plugin-msteams/server/msteams/clientmodels"
-	mockClient "github.com/mattermost/mattermost-plugin-msteams/server/msteams/mocks"
-	mockStore "github.com/mattermost/mattermost-plugin-msteams/server/store/mocks"
 	"github.com/mattermost/mattermost-plugin-msteams/server/store/storemodels"
-	"github.com/mattermost/mattermost-plugin-msteams/server/testutils"
 	"github.com/mattermost/mattermost/server/public/model"
-	"github.com/mattermost/mattermost/server/public/plugin/plugintest"
 	"github.com/pkg/errors"
 	"golang.org/x/oauth2"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
 )
 
+func assertNoCommandResponse(t *testing.T, actual *model.CommandResponse) {
+	t.Helper()
+
+	require.NotNil(t, actual)
+	require.Equal(t, &model.CommandResponse{}, actual)
+}
+
+func assertEphemeralResponse(th *testHelper, t *testing.T, args *model.CommandArgs, message string) {
+	t.Helper()
+
+	th.assertEphemeralMessage(t, args.UserId, args.ChannelId, message)
+}
+
```

Comment:
I would love to see these helper methods in a shared repo, potentially https://github.com/mattermost/mattermost/tree/master/server/public/pluginapi, so other plugins can reuse them. Same goes for the `testHelper`.

Reply:
Yes, 100%. I wanted to let these soak a bit, but also there's the problem that the `public/` module can't reimport the `server/v8` module that's currently required to spin up the server. Need a larger solution (which I intend to eventually tackle!)